utils/utils.go

```diff
@@ -61,4 +63,11 @@ func GetPokemonByKey(params url.Values, pokemonList model.PokemonList) model.Pok
 	}
 
 	return filteredPokemon
-}
\ No newline at end of file
+}
+
+/*
+GetIntFromDivision returns an int from a division of two ints
+*/
+func GetIntFromDivision(x int, y int) int {
```

Comment:
Remember that you can make this line a bit shorter by grouping common data types -> `(x, y int)`

Reply:
💯